test/ExceptionTest.php

```diff
@@ -18,9 +17,13 @@
 
 class ExceptionTest extends TestCase
 {
-    public function exception() : Generator
+    /**
+     * @psalm-return iterable<string, class-string[]>
+     * @psalm-suppress MoreSpecificReturnType
```

Comment:
This suppression looks suspicious: is it somehow fixable? Is this an inherited method?

Reply:
This is because parser does not know that `$namespace . $class` is `class-string`, line 32. 
Not sure how else I should specify that type.

Reply:
Ah, that's solved by asserting that `class_exists()` on the resulting string